### PR TITLE
- [zh] Translate pod-template-v1.md

### PR DESCRIPTION
**Pull request by qlijin**
fix issue #34270

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 1]
```diff
@@ -0,0 +1,906 @@
...
+<hr>
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  <!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+  标准对象的元数据。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **spec** (<a href="{{< ref "../workload-resources/pod-v1#PodSpec" >}}">PodSpec</a>)
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:89]: We need to translate this page first, or the build won't be success.

**Review by Sea-n — COMMENTED** [review]
Please refer to [Netlify deploy log])https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/62aab6646ccdb000085783e1#L77).

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md (hunk 1 above)
Comment (Sea-n) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:89]: Waiting for #34294.
- #34294

Comment (qlijin) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:89]: OK.  This page's build relies on other pages

Comment (qlijin) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:89]: The docs/reference/kubernetes-api/workload-resources/pod-v1.md also relies:
docs/reference/kubernetes-api/config-and-storage-resources/volume.md
docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md

these two file is in #https://github.com/kubernetes/website/issues/34158

Comment (windsonsea) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:89]: Yes, I need more time to handle `config-and-storage-resources/volume.md` that is relatively big with nearly 6000 words.

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 2]
```diff
@@ -0,0 +1,906 @@
+---
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "PodTemplate"
+content_type: "api_reference"
+description: "PodTemplate 描述了一种创建预定义 pod 副本的模板。"
+title: "PodTemplate"
+weight: 2
+auto_generated: true
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:10]: remove this line 10

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 3]
```diff
@@ -0,0 +1,906 @@
...
+  import: "k8s.io/api/core/v1"
+  kind: "PodTemplate"
+content_type: "api_reference"
+description: "PodTemplate describes a template for creating copies of a predefined pod."
+title: "PodTemplate"
+weight: 2
+auto_generated: true
+-->
+
+<!--
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:25]: Line 25-34 can be removed. en is generated but zh is localized.

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 4]
```diff
@@ -0,0 +1,906 @@
...
+
+`import "k8s.io/api/core/v1"`
+
+
+## PodTemplate {#PodTemplate}
+
+<!--
+PodTemplate describes a template for creating copies of a predefined pod.
+-->
+PodTemplate 描述了一种创建预定义 pod 副本的模板。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:46]:
```suggestion
PodTemplate 描述了一种创建预定义 Pod 副本的模板。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 5]
```diff
@@ -0,0 +1,906 @@
...
+To update the reference content, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+`apiVersion: v1`
+
+`import "k8s.io/api/core/v1"`
+
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:39]: Only one empty line is enough. You can remove it if two or more empty lines exist.

**Review by windsonsea — CHANGES_REQUESTED** [review]
Only viewed a beginging part. May need much change and wait for other files to merge.

**Comment by qlijin** [comment]
ok. There will be a new commit

**Comment by tengqm** [comment]
Which file is required by this one?

**Comment by qlijin** [comment]
> Which file is required by this one?

Wait for volume.md and job-v1.md.   
persistent-volume-claim-v1.md is done.

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 6]
```diff
@@ -0,0 +1,906 @@
...
+-->
+#### 参数
+
+
+<!--
+- **name** (*in path*): string, required
+
+  name of the PodTemplate
+-->
+- **name** (**位于路径中**): string,必需
```
Comment (zhangxyjlu) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:166]:
```suggestion
- **name** (**路径参数**): string,必需
```

Comment (zhangxyjlu) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:166]: 其他地方也排查一下

Comment (qlijin) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:166]: OK

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 7]
```diff
@@ -0,0 +1,906 @@
+---
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "PodTemplate"
+content_type: "api_reference"
+description: "PodTemplate 描述了一种创建预定义 pod 副本的模板。"
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:7]: 正文中出现 pod 时，一律改拼写为 Pod，符合中英文混排习惯。

```suggestion
description: "PodTemplate 描述了一种模板，用来为预定义的 Pod 生成副本。"
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 8]
```diff
@@ -0,0 +1,906 @@
...
+
+- **kind**: PodTemplate
+
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  <!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+  标准对象的元数据。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:61]:
```suggestion
  标准的对象元数据。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 9]
```diff
@@ -0,0 +1,906 @@
...
+
+- **template** (<a href="{{< ref "../workload-resources/pod-template-v1#PodTemplateSpec" >}}">PodTemplateSpec</a>)
+
+  <!--
+  Template defines the pods that will be created from this pod template. https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+  -->
+  Template 定义将从这个 pod template 创建的pod。 https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:68]: 字段名称是 template，首字母要改为小写。英文的不准确。可参考其他译文。
中英文之间留白。请彻查。

```suggestion
  template 定义将从这个 Pod 模板 创建的 Pod。 https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 10]
```diff
@@ -0,0 +1,906 @@
...
+
+
+
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:71]: 如上空行只需保留一个。请彻查。

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 11]
```diff
@@ -0,0 +1,906 @@
...
+## PodTemplateSpec {#PodTemplateSpec}
+
+<!--
+PodTemplateSpec describes the data a pod should have when created from a template
+-->
+PodTemplateSpec 描述了从模板创建pod时应具有的数据
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:77]:
```suggestion
PodTemplateSpec 描述从模板创建的 Pod 所应具有的数据。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 12]
```diff
@@ -0,0 +1,906 @@
...
+
+
+<hr>
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  <!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+  标准对象的元数据。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:87]:
```suggestion
  标准的对象元数据。更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 13]
```diff
@@ -0,0 +1,906 @@
...
+
+- **spec** (<a href="{{< ref "../workload-resources/pod-v1#PodSpec" >}}">PodSpec</a>)
+
+  <!--
+  Specification of the desired behavior of the pod. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+  -->
+  pod 所需行为的规范。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:94]:
```suggestion
  Pod 预期行为的规约。更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 14]
```diff
@@ -0,0 +1,906 @@
...
+
+- **kind**: PodTemplateList
+
+
+- **metadata** (<a href="{{< ref "../common-definitions/list-meta#ListMeta" >}}">ListMeta</a>)
+
+  <!--
+  Standard list metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+  -->
+  标准列表元数据, 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:120]:
```suggestion
  标准的列表元数据。更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 15]
```diff
@@ -0,0 +1,906 @@
...
+  <!--
+  List of pod templates
+  -->
+  PodTemplate 列表
+
+
+<!--
+## Operations {#Operations}
+
+<hr>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:136]: 删除135-136,140，143 空行

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 16]
```diff
@@ -0,0 +1,906 @@
...
+-->
+- **name** (**位于路径中**): string,必需
+
+  PodTemplate 的名称
+
+
+<!--
+- **namespace** (*in path*): string, required
+-->
+- **namespace** (**位于路径中**): string,必需
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:174]: 请彻查。

```suggestion
- **namespace** (**位于路径中**)：string，必需
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md [hunk 17]
```diff
@@ -0,0 +1,906 @@
...
+
+401: Unauthorized
+
+
+<!--
+### `list` list or watch objects of kind PodTemplate
+
+#### HTTP Request
+-->
+### `list` 列出或监控 PodTemplate 类型的对象
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md:204]:
```suggestion
### `list` 列出或监视 PodTemplate 类型的对象
```

**Comment by qlijin** [comment]
已按照建议修改，但还是依赖pod-v1.md

**Comment by tengqm** [comment]
> 已按照建议修改，但还是依赖pod-v1.md #34630 请注意是pod-v1.md 不是job-v1.md

This one depends on the pod-v1.md file, which was claimed by @ydFu ten days ago. We haven't got a draft to review.
There is no dependency on 34630.

**Comment by Sea-n** [comment]
Wait for `pod-v1.md`.
- #35024

**Comment by tengqm** [comment]
Please rebase this onto 'main' branch where 'pod-v1.md' has been translated.

**Comment by tengqm** [comment]
Since this is blocking many other API references, I'm gonna take it over and propose a new PR.
/close

**Comment by qlijin** [comment]
OK

**Pull request closed without merging**